src/vivarium_census_prl_synth_pop/results_processing/generate_names.py

```diff
@@ -48,16 +48,14 @@ def fromat_data_for_name_generation(
     # Get year of birth - string column and not timestamp
     data["year_of_birth"] = data["date_of_birth"].str[:4]
 
+    # We only need one dataframe here since first_name_id and middle_name_id will be the same.
```

Comment:
Let's not rely on this, since it could change in the future.

Reply:
That is fine but will require some updates to the name generator and related tests.